src/libs/Navigation/AppNavigator/createCustomStackNavigator/index.js

```diff
@@ -25,6 +29,31 @@ const defaultProps = {
     screenOptions: undefined,
 };
 
+const IFRAME_CONSTANT_KEY = 'iframe-constant-key';
+
+function splitRoutes(routes) {
+    const iframeRoutes = [];
+    const rhpRoutes = [];
+    const otherRoutes = [];
+
+    routes.forEach((route) => {
+        const topMostRouteName = getTopMostCentralPaneRouteName({routes: [route]});
+
+        if (route.name === NAVIGATORS.CENTRAL_PANE_NAVIGATOR && topMostRouteName !== SCREENS.REPORT) {
```

Comment:
this model of central panes being nested inside on another is admittedly very confusing... (although we gonna probably refactor this soon)

so for the iframe pages, is it inside another central pane route?